Library/Formula/pius.rb

```diff
@@ -10,7 +11,7 @@ def install
     ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
     # Replace hardcoded gpg path (WONTFIX):
     # https://sourceforge.net/p/pgpius/bugs/12/
-    inreplace "libpius/constants.py", "/usr/bin/gpg", "#{HOMEBREW_PREFIX}/bin/gpg"
+    inreplace "libpius/constants.py", %r{/usr/bin/gpg2?}, "#{HOMEBREW_PREFIX}/bin/gpg"
```

Comment:
This may not always be correct, since if you have `gpg2` installed but not `gpg` it won't force the latter installation and there won't be a `gpg` binary in the `$PATH`. We may need to invent some logic to handle this situation. Thoughts @xu-cheng?